openreview/conference/templates/recruitResponseWebfield.js

```diff
@@ -34,8 +35,30 @@ function render() {
           '</div>',
         '</div>'
       ].join('\n'));
+    } else if (declined) {
+      // Get invitation to request max load
+      var reduced_load_invitation_id = CONFERENCE_ID + '/-/' + REDUCED_LOAD_INVITATION_NAME;
+      Webfield.get('/invitations', { regex: reduced_load_invitation_id })
+      .then(function(result) {
+        if (result.hasOwnProperty('invitations') && result.invitations.length) {
+          invitation = result.invitations[0];
+          $response.append([
+            '<div class="panel">',
+              '<div class="row">',
+                '<strong><p>If you declined because you are looking for a reduced load, please use this <strong><a href="/invitation?id=' + reduced_load_invitation_id + '&invitation_response=Yes&user=' + args.user + '&key=' + args.key + '">LINK</a></strong> to request a load you would be comfortable reviewing.</p></strong>',
```

Comment:
I would change the text to, 
```
'<p><strong>You have declined the invitation from ' + HEADER.title + '.</strong>' +
'<p><strong>If you chose to decline because the number of reviews required was too high, you can request a reduced reviewer load by clicking here: <a>Request reduced load</a></strong></p>'
```

Reply:
But actually I think we should use `h3`s instead of `p` and `strong`

Reply:
Can we add that first line "You have declined..." in too?